src/components/pdf-thumbnails-sidebar/PDFThumbnailSidebar.scss

```diff
@@ -1,77 +1,63 @@
+@import "../bootstrap/core";
+
 .pdf-thumbnails-view {
   position: absolute;
   top: 0;
-  bottom: 0;
-  padding: 10px 30px 0;
-  overflow: auto;
-  -webkit-overflow-scrolling: touch;
+  left: 0;
+  box-sizing: border-box;
   width: 100%;
-
-  > a:active,
-  > a:focus {
-    outline: 0;
-  }
+  height: 100%;
+  padding: 1rem;
+  overflow-x: hidden;
+  overflow-y: auto;
 
   .thumbnail {
-    margin: 0 10px 5px 10px;
-    float: left;
+    margin: 0 0 1rem;
     &:not([data-loaded]) {
       border: 1px dashed rgba(255, 255, 255, 0.5);
       margin: -1px 9px 4px 9px;
     }
 
     &.selected {
-      > .thumbnailSelectionRing {
-        background-color: hsla(0,0%,100%,.3);
-        background-image: linear-gradient(hsla(0,0%,100%,.05), hsla(0,0%,100%,0));
-        background-clip: padding-box;
-        box-shadow: 0 1px 0 hsla(0,0%,100%,.05) inset,
-        0 0 1px hsla(0,0%,100%,.1) inset,
-        0 0 1px hsla(0,0%,0%,.2);
-        color: hsla(0,0%,100%,1);
-
-        > .thumbnailImage {
-          box-shadow: 0 0 0 1px hsla(0,0%,0%,.5);
-          opacity: 1;
-        }
+      .thumbnailSelectionRing {
+        box-shadow: 0 0 0 4px rgba($primary, 0.5);
       }
     }
-
   }
 
   > a:last-of-type > .thumbnail {
     margin-bottom: 10px;
   }
 
   .thumbnailSelectionRing {
+    width: auto !important;
+    height: auto !important;
     border-radius: 2px;
-    padding: 7px;
-    display: inline-table;
+    transition: box-shadow 150ms;
   }
 
   .thumbnailImage {
-    border: 1px solid transparent;
-    box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.5), 0 2px 8px rgba(0, 0, 0, 0.3);
-    opacity: 0.8;
-    z-index: 99;
+    width: 100% !important;
+    height: auto !important;
+    border: 1px solid $gray-400;
+    border-radius: 2px;
     background-color: white;
-    background-clip: content-box;
+    box-shadow: 0 1px 2px rgba(0, 0, 0, 0.15);
   }
 
-
-  a:focus > .thumbnail > .thumbnailSelectionRing > .thumbnailImage,
-  .thumbnail:hover > .thumbnailSelectionRing > .thumbnailImage {
-    opacity: .9;
-  }
-
-  a:focus > .thumbnail > .thumbnailSelectionRing,
-  .thumbnail:hover > .thumbnailSelectionRing {
-    background-color: hsla(0,0%,100%,.15);
-    background-image: linear-gradient(hsla(0,0%,100%,.05), hsla(0,0%,100%,0));
-    background-clip: padding-box;
-    box-shadow: 0 1px 0 hsla(0,0%,100%,.05) inset,
-    0 0 1px hsla(0,0%,100%,.2) inset,
-    0 0 1px hsla(0,0%,0%,.2);
-    color: hsla(0,0%,100%,.9);
-  }
+  // a:focus > .thumbnail > .thumbnailSelectionRing > .thumbnailImage,
```

Comment:
You can completely remove these lines of code

Reply:
Thank you! I forgot to clean-up in the evening.